libs/net/kafka/KafkaListener`.cs

```diff
@@ -193,20 +216,57 @@ public async Task ConsumeAsync(Func<ConsumeResult<TKey, TValue>, Task<ConsumerAc
         }
         catch (Exception ex)
         {
-            // https://github.com/edenhill/librdkafka/blob/master/INTRODUCTION.md#fatal-consumer-errors
-            // An unhandled exception will stop consuming as I don't know of a way to resume the paused consumer.
-            _logger.LogError(ex, "Error while consuming. {message}", ex.Message);
-            proceed = this.OnError?.Invoke(this, new ErrorEventArgs(ex)) ?? ConsumerAction.Stop;
+            proceed = HandleException(ex);
         }
         finally
         {
-            if (proceed == ConsumerAction.Stop) this.Stop();
-            else if (this.IsPaused && proceed == ConsumerAction.Proceed)
+            if (!UseMultiThreads) FinalCleanUp(proceed);
+        }
+    }
+
+    private async Task<ConsumerAction> RunActionAsync(
+        Func<ConsumeResult<TKey, TValue>, Task<ConsumerAction>> action,
+        ConsumeResult<TKey, TValue>? _currentResult)
```

Comment:
Convention, never name an argument with an `_` underscore.  It is misleading on its intent.

Reply:
Updated, thanks